src/meta/src/hummock/compaction/picker/min_overlap_compaction_picker.rs

```diff
@@ -112,6 +115,16 @@ impl CompactionPicker for MinOverlappingPicker {
         level_handlers: &[LevelHandler],
         stats: &mut LocalPickerStatistic,
     ) -> Option<CompactionInput> {
+        let mut picker =
+            TrivialMovePicker::new(self.level, self.target_level, self.overlap_strategy.clone());
+        if let Some(input) = picker.pick_trivial_move_task(
+            &levels.get_level(self.level).table_infos,
+            &levels.get_level(self.target_level).table_infos,
+            level_handlers,
+            stats,
+        ) {
+            return Some(input);
```

Comment:
If there is always some SST that can be trivial-moved, will early-return here be a problem?

Reply:
It is still right. Because Trivial Move task always has the smallest score(which is zero). So `MinOverlappingPicker` will still return the SST that can be trivial-moved at first.